chautari-rental-service.yml

```diff
@@ -0,0 +1,938 @@
+components:
+  securitySchemes:
+    basicAuth:
+      type: http
+      scheme: basic
+  schemas:
+    ErrorModel:
+      type: object
+      properties:
+        developerMessage:
+          type: string
+          description: >-
+            Verbose, plain language description of the problem for the app
+            developer with hints about how to fix it.
+        userMessage:
+          type: string
+          description: Pass this message on to the app user if needed
+        errorCode:
+          type: string
+          description: Error code defined for the error
+        moreInfo:
+          type: string
+          description: Any additional information on the error (e.g. website links)
+    Credential:
+      type: object
+      required:
+        - userName
+        - password
+      properties:
+        userName:
+          type: string
+        password:
+          type: string
+        security-questions:
+          type: array
+          items:
+            $ref: '#/components/schemas/SecurityQuestionList'
+    SecurityQuestionList:
+      type: object
+      properties:
+        question:
+          type: string
+        answer:
+          type: string
+    AuthenticationResponse:
+      type: object
+      properties:
+        message:
+          type: string
+    UserAccountDetail:
+      type: object
+      properties:
+        user-type:
+          type: string
+          enum: [renter, owner, both]
+        user:
+          $ref: '#/components/schemas/UserDetail'
+        credential:
+          $ref: '#/components/schemas/Credential'
+        createdOn:
+          type: string
+    UserDetail:
+      type: object
+      required:
+        - person
+      properties:
+        person:
+          $ref: '#/components/schemas/PersonName'
+        address:
+          $ref: '#/components/schemas/Address'
+    Address:
+      type: object
+      properties:
+        address1:
+          type: string
+        address2:
+          type: string
+        city:
+          type: string
+        state:
+          type: string
+        country:
+          type: string
+        zip:
+          type: integer
+    RentalItemResponse:
+      type: object
+      required:
+        - rental-id
+      properties:
+        rental-id:
+          type: string
+
+    PersonName:
+      type: object
+      properties:
+        title:
+          type: string
+          enum: [Ms, Mr, Miss]
+        firstName:
+          type: string
+        middleName:
+          type: string
+        lastName:
+          type: string
+        status:
+          type: string
+          enum: [single, married]
+    SetupResponse:
+      type: object
+      properties:
+        user-id:
+          type: string
+          format: UUID
+        message:
+          type: string
+    RentalItemRequest:
+      type: object
+      required:
+        - saf
+      properties:
+        owner-id:
+          type: string
+        title:
```

Comment:
what is title for?

Reply:
this is the title of the rental item. like -- Two-room at the lakeside